CMakeLists.txt

```diff
@@ -108,6 +107,11 @@ if(WITH_CUDA)
   endif()
 
   set(CUDA_NVCC_FLAGS         "${CUDA_NVCC_FLAGS};--default-stream;per-thread")
+
+  if(CCTAG_USE_POSITION_INDEPENDENT_CODE)
```

Comment:
u can just check `BUILD_SHARED_LIBS` instead of using another variable

Reply:
Yes, that's the idea of removing `COMPILE_STATIC_CCTAG_LIBRARY` but position independent is a bit different.
I'm currently using the static build with position independent code to link [AliceVision](https://github.com/alicevision/AliceVision).